packages/cli/scripts/dev.js

```diff
@@ -0,0 +1,38 @@
+const { handlerFactory, createWebpackConfig } = require('../lib/cliUtils');
+const { watchCompiler, handleCompilationError } = require('../lib/compiler');
+const { info } = require('../lib/print');
+const { SharedArgs } = require('../constants');
+
+async function dev(args) {
+  try {
+    info('Parsing webpack configuration file...');
+    const config = await createWebpackConfig(['client', 'server'], {
+      ...args,
+      isProduction: false,
+      isWatch: true
+    });
+
+    info('Starting webpack compiler...');
+    await watchCompiler(config, args);
+  } catch (err) {
+    console.log(err);
```

Comment:
Why do you use log and handleCompilationError method?

Reply:
Fixed